File: 750A-NewYearAndHurry/750a-newYearAndHurry.c
#include <stdio.h>
int main(){
	int n, k;
	scanf("%d  %d",&n,&k);
	int org_n = n, time_left = 240 - k;
	int i = 1;
	while(time_left >= 0 && n > 0){
		time_left -= 5*i;
		if(time_left < 0)
			break;
		n--;
		i++;
	}
	printf("%d",org_n-n);
}

File: 750A-NewYearAndHurry/750a-newYearAndHurry.py
p = list(map(int, input().split()))
n = p[0]
org_n = n
k = p[1]
time_left = 240 - k
i = 1
while time_left >= 0 and n > 0:
    time_left -= 5*i
    if time_left < 0:
        break
    n -= 1
    i += 1
print(org_n - n)
